LayoutTests/svg/filters/feComponentTransfer-gamma.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="fuzzy" content="maxDifference=0-172; totalPixels=0-36515" />
+    <title>Gamma Filter</title>
+    <style>
+        img {
+            max-width: 500px;
+            filter: url(#gamma-filter);
+        }
+
+        .gamma-svg {
+            position: absolute;
+            width: 0;
+            height: 0;
+            pointer-events: none;
+        }
+    </style>
+</head>
+<body>
+    <img id="original-image" class="demo-image" src="resources/room.jpg">
```

Comment:
Do we have to use image to write this test? Can't we just apply the filter to a rectangle fill with solid color?

Reply:
What is `demo-image`?